bodhi/server/metadata.py

```diff
@@ -306,3 +306,31 @@ def cache_repodata(self):
             shutil.rmtree(cache)
         shutil.copytree(repodata, cache)
         log.info('%s cached to %s' % (repodata, cache))
+
+
+class PungiMetadata(ExtendedMetadata):
+
+    def __init__(self, release, request, db, path, compose_dir):
+        super(PungiMetadata, self).__init__(release, request, db, path)
+        self.compose_dir = compose_dir
+
+    def modifyrepo(self, filename):
```

Comment:
WARNING!!! This WILL result in totally unusable repositories if you try to insert *ANY* file other than "updateinfo.xml". This function hardcodes updateinfo for the type to be inserted into the repomd, so whatever file you try to insert with this overrides updateinfo.xml!

Reply:
https://bodhi.fedoraproject.org/docs/release_notes.html#id61

Reply:
I would *strongly* suggest to just use the one already there, since that will be fixed with #1876.